src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription(){
+        return this.description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]"); //return tick or X symbols
+    }
+
+    public void setDone(){
+        this.isDone = true;
+    }
+
+    public String toString(){
+        return getStatusIcon() + " " + getDescription();
+    }
+}
```

Comment:
Hi good job on the codes.
If I remember correctly, Prof mentioned that it's better to leave an empty line at the of all the files :)